scripts/postInstall.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Go to project root
+ROOT_DIR=$(dirname "$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &> /dev/null && pwd)")
```

Comment:
Source: https://stackoverflow.com/questions/59895/how-do-i-get-the-directory-where-a-bash-script-is-located-from-within-the-script